pipeline/scripts/4/2/tier-2/test-deploy_okr_bz_1994827.sh

```diff
@@ -0,0 +1,49 @@
+#! /bin/sh
+echo "Begin testing of Red Hat Ceph RPM based upgrade."
+
+random_string=$(cat /dev/urandom | tr -cd 'a-z0-9' | head -c 5)
+instance_name="ci-${random_string}"
+platform="rhel-7"
+rhbuild="4.2"
+test_suite="suites/nautilus/upgrades/tier-2_okr_bz_1994827.yaml"
+test_conf="conf/nautilus/upgrades/tier-2_okr_BZ_1994827.yaml"
+test_inventory="conf/inventory/rhel-7.9-server-x86_64.yaml"
```

Comment:
Let's start using the soft link that points to the latest i.e. `rhel-7-latest.yaml`